### DISCUSSION TITLE
v7.3 - Préparation à la génération des packages

### DISCUSSION THREAD

**Discussion by smdsgn** [MTES-MCT/sparte#539] [Recettes]
## Fonctionnalités
* Création de l'onglet imperméabilisation dans https://github.com/MTES-MCT/sparte/pull/493
* Renommer "renaturation" en "désartificialisation" dans https://github.com/MTES-MCT/sparte/pull/513
* Remplacer les termes "déterminants" et "non renseigné" dans https://github.com/MTES-MCT/sparte/pull/511
* Ajout d'un message pour indiquer la non utilisation des seuils dans le calcul de l'artif dans https://github.com/MTES-MCT/sparte/pull/510
* Migration des tableaux de données vers la nouvelle version du dsfr dans https://github.com/MTES-MCT/sparte/pull/528

## Correctifs
* Ajout de la consommation de la commune aux tableaux de comparaison de la consommation (relative et absolue) dans https://github.com/MTES-MCT/sparte/pull/490
* Amélioration du contraste des emprises sur le fond de carte OpenStreetMap dans https://github.com/MTES-MCT/sparte/pull/514
* Corrections de typos et indication de la disponibilité de l'imperméabilisation dans le rapport local dans https://github.com/MTES-MCT/sparte/pull/509
* Prévenir l'accès aux pages qui ont besoin de données OCS GE si le territoire n'a pas d'OCS GE dans https://github.com/MTES-MCT/sparte/pull/520
* Remplacement des données absolues des cartes choroplèthes par des données relatives dans https://github.com/MTES-MCT/sparte/pull/526
* Correction du nom de la carte de présentation des zonages d'urbanisme dans https://github.com/MTES-MCT/sparte/pull/534
* Correction mauvaise conso totale et annuelle de la projection zan dans l'export dans https://github.com/MTES-MCT/sparte/pull/524

**Liste complète des changements**: https://github.com/MTES-MCT/sparte/compare/v7.2...v7.3

<hr /><em>This discussion was created from the release <a href='https://github.com/MTES-MCT/sparte/releases/tag/v7.3'>Préparation à la génération des packages</a>.</em>

**Comment by alexisig** [comment]
<img width="571" alt="image" src="https://github.com/user-attachments/assets/9d92ae49-a393-4abd-af62-c19540ecc309">
